content/resources/user/data-types.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Gen3 - Types of Data"
+date: 2019-01-14T15:09:06-05:00
+linktitle: /resources/user
+layout: withtoc
+menuname: userMenu
+---
+{{% markdownwrapper %}}
+* * *
+## 1. Understanding Data Representation in Gen3
+
+* * *
+In a Gen3 data commons, a semantic distinction is made between two types of data: "data files" and "metadata".
+
+* A "data file" would be information like tabulated data values in spreadsheet or a fastq/bam file containing DNA sequences. The contents of the file are not exposed to the API as queryable properties, so the file must be downloaded to view its content.
+
+* "Metadata" are variables or key/value pairs that can be queried via the Gen3 data commons' API or viewed in the Gen3 data commons' data exploration tool. In this sense, a key/value pair is essentially a variable and its value. In a Gen3 data dictionary, variable names are termed "properties", and data contributors provide the values for these pre-defined properties in their data submissions.
```

Comment:
I gather some some submitters are highly technical but I wonder if more abstract terminology would be helpful to the average user for understanding the data/metadata duality, maybe something like "properties that help to organize or convey additional information about corresponding data files" vs variables or keys/values which comes off as kind of technical. This is more of a philosophical question really, I know the whole subject of data/metadata is different depending on who you ask and it's hard to describe our system and usage concisely to new users. Just my 2 cents